DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -129,35 +106,38 @@ juju config kratos-external-idp-integrator \
   microsoft_tenant_id=<tenant_id>
 ```
 
-After a while the *kratos-external-idp-integrator* status will change to active, this means that Azure AD has been added as a Kratos sign in provider.
+> See more: [Charmhub | Kratos External Idp Integrator](https://charmhub.io/kratos-external-idp-integrator)
+
+After a while the kratos-external-idp-integrator status will change to active, this means that Azure AD has been added as a Kratos sign in provider. Congratulations, your deployment now uses Azure AD as your external identity provider.
 
-# Add SSO to Charmed Grafana 
+# Use Identity Platform to provide SSO
 
-To deploy [Charmed Grafana](https://charmhub.io/grafana-k8s), run:
+Now that our Identity Broker is ready, we can use it to provide SSO to an application. For this purpose we are going to use Grafana. To deploy [Charmed Grafana](https://charmhub.io/grafana-k8s), run:
 
 ```
 juju deploy grafana-k8s
 ```
 
-This command deploys Charmed Grafana in the same model as the Identity Platform Bundle.
-
-The newly deployed Grafana requires ingress, and it can get that from the existing Traefik instance that was deployed together with the bundle. 
+This command deploys Charmed Grafana in the same model as the Identity Platform bundle.
 
+The newly deployed Grafana requires ingress in order to properly integrate with the Identity Platform. We can use the existing Traefik instance, that was deployed together with the bundle, to get that.
 You can integrate Charmed Grafana with Traefik by running:
 
 ```
 juju integrate grafana-k8s:ingress traefik-public
 ```
 
-And then you can integrate Grafana with Hydra, by running:
+Now, you can connect Grafana with the bundle. To do this we need to integrate Grafana with the bundle’s OIDC server, Hydra:
 
 ```
 juju integrate grafana-k8s:oauth hydra
 ```
 
-This command registers Grafana as an OIDC client in Hydra and configures Grafana to use the Canonical Identity Platform as an authentication provider.
+This command registers Grafana as an OIDC client in Hydra and configures Grafana to use the Canonical Identity Platform as an authentication provider. Once all applications are ready we can try to login in to Grafana through Azure AD.
```

Comment:
Add comma, drop hedge ("we can try to"), fix spelling (login -> log in, because verb):

> Once all applications are ready, log in to Grafana through Azure AD.